classify/adaptmatch.cpp

```diff
@@ -2242,7 +2242,7 @@ void Classify::ShowBestMatchFor(int shape_id,
   tprintf("Static Shape ID: %d\n", shape_id);
   ShowMatchDisplay();
   im_.Match(ClassForClassId(PreTrainedTemplates, shape_id),
-            AllProtosOn, reinterpret_cast<BIT_VECTOR>(&config_mask),
+            AllProtosOn, &config_mask, // TODO: or reinterpret_cast<BIT_VECTOR>(&config_mask) anyway?
```

Comment:
I don't know what the issues are here, so it seemed presumptuous to just remove all traces of the cast as if I did. Currently there's no need for a cast, but then why was it there in the first place?